kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -42,11 +42,11 @@ data:
           # When the setting is set to false, only the kube events with !normal event type will be collected
           enabled = false
           # When this is enabled (enabled = true), all kube events including normal events will be collected
-       #[log_collection_settings.schema]
+       [log_collection_settings.schema]
           # In the absence of this configmap, default value for containerlog_schema_version is "v1"
```

Comment:
we should rephrase this statement from 
# In the absence of this configmap, default value for containerlog_schema_version is "v1"
To
# In the absence of this configmap, default value for containerlog_schema_version is "v1" if v2 not enabled through onboarding